migration/src/main/resources/META-INF/rewrite/axon-jakarta.yml

```diff
@@ -0,0 +1,74 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.axonframework.axon.UpgradeAxonFramework_4_7_Jakarta
+displayName: Upgrade to Axonframework 4.7 Jakarta
+description: This is what you will want if you also upgrade to Spring 6, Spring Boot 3, or another framework that depends on jakarta packages.
+recipeList:
+  # Convert to Spring Boot 3, Spring FrameWork 6, Jakarta packages, JUnit 5, Java 17, etc.
+  #TODO - org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_0
+  # Swap 4.6 dependencies for 4.7 dependencies
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-configuration-jakarta
+      newArtifactId: axon-configuration
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-eventsourcing-jakarta
+      newArtifactId: axon-eventsourcing
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-messaging-jakarta
+      newArtifactId: axon-messaging
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-modelling-jakarta
+      newArtifactId: axon-modelling
+      newVersion: 4.7.x
+  # Convert legacy JPA packages to Jakarta JPA packages
+  - org.openrewrite.java.ChangeType:
```

Comment:
I don't think this section is necessary, although I may not understand the OpenRewrite standard thoroughly enough. 
So, bear with me.

If a user upgrades from AF 4.6 to AF 4.7, and at the same time they're moving to the Jakarta namespace, they can stick to the current packages of Axon's JPA-specific infrastructure components.

So, that would make these steps _only_ worthwhile when a user is already on AF 4.7, but stuck to Javax initially.

Reply:
Then again, I believe rules are ignored if they do not apply, right? 
So, perhaps, these are here to serve that exact purpose I described in the previous comment: users already on AF 4.7, that want to migrate from Javax (our "new" legacy packages) to Jakarta.

Reply:
Correct; these are here for users that first migrated to 4.7 without Jakarta, and then want to switch to Jakarta later. It was easy enough to add, and should help users wherever they are in their migration.

Reply:
Awesome. Then it's totally reasonable to contain these steps here.